packages/@glimmer/debug/lib/stack-check.ts

```diff
@@ -234,7 +234,7 @@ class ObjectChecker implements Checker<unknown> {
   type!: object;
 
   validate(obj: unknown): obj is object {
-    return typeof obj === 'object' && obj !== null;
+    return typeof obj === 'function' || (typeof obj === 'object' && obj !== null);
```

Comment:
Originally was using this and realized it was incorrect, so I fixed it. Ended up not actually needing it, but wanted to make sure it was fixed.